OpenDreamRuntime/Objects/Types/DreamObjectAtom.cs

```diff
@@ -5,7 +5,6 @@ namespace OpenDreamRuntime.Objects.Types;
 
 [Virtual]
 public class DreamObjectAtom : DreamObject {
-    public string? Name;
     public string? Desc;
```

Comment:
```suggestion
```

could probably clip this also?

Reply:
Ohh, yep I put that one back in to find all the usages of it when it errored and forgot to remove it again